algorithmic_efficiency/workloads/cifar/cifar_pytorch/workload.py

```diff
@@ -0,0 +1,248 @@
+"""CIFAR10 workload implemented in PyTorch."""
+
+import contextlib
+import math
+import os
+import random
+from typing import Dict, Tuple
+
+import torch
+from torch import nn
+import torch.distributed as dist
+import torch.nn.functional as F
+from torch.nn.parallel import DistributedDataParallel as DDP
+from torchvision import transforms
+from torchvision.datasets import CIFAR10
+
+from algorithmic_efficiency import data_utils
+from algorithmic_efficiency import param_utils
+from algorithmic_efficiency import spec
+import algorithmic_efficiency.random_utils as prng
+from algorithmic_efficiency.workloads.cifar.workload import BaseCifarWorkload
+from algorithmic_efficiency.workloads.imagenet.imagenet_pytorch.models import \
+    resnet18
+
+PYTORCH_DDP = 'LOCAL_RANK' in os.environ
```

Comment:
Could you change `PYTORCH_DDP` to `USE_PYTORCH_DDP`?

Reply:
Fixed!